frontend/libs/studio-content-library/src/ContentLibrary/LibraryBody/pages/CodeListPage/CodeListsActionsBar/AddCodeListDropdown/utils.ts

```diff
@@ -0,0 +1,31 @@
+import type { ExternalResource } from 'app-shared/types/ExternalResource';
+import { LibraryContentType } from 'app-shared/enums/LibraryContentType';
+import { StringUtils } from '@studio/pure-functions';
+
+export const getCodeListIdsFromExternalResources = (
+  externalResourceIds: ExternalResource[],
```

Comment:
Keep the naming convention the same as in components. The same change will be for a few variables in this file
```suggestion
  externalResources: ExternalResource[],
```